patches/chrome-common-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
+index b45794f858d4f8eabceab39166f2103dbdf87361..df4fc33bae12877dec8724a6ea2bde4ef649ea5e 100644
+--- a/chrome/common/BUILD.gn
++++ b/chrome/common/BUILD.gn
+@@ -536,6 +536,7 @@ static_library("non_code_constants") {
+     "//printing/buildflags",
+     "//ui/base:ui_features",
+   ]
++  if (brave_chromium_build) { configs += [ "//brave/common:constants_configs" ] }
```

Comment:
Added to use `BRAVE_PRODUCT_STRING` define in `chrome_constants.cc`.
This define has different string for different channel.(Brave-Browser, Brave-Browser-Nightly)

Reply:
see comment below